doc-content/enterprise-only/admin-and-config/managing-business-central-configuring-githooks-proc.adoc

```diff
@@ -1,63 +1,63 @@
 [id='managing-business-central-configuring-githooks-proc']
-= Configuring git hooks
+= Configuring Git hooks
 
-{CENTRAL} can automatically push changes to a remote repository using Git hooks. After you configure the post-commit hook, {CENTRAL} automatically pushes your changes to the remote repository.
+You can use {CENTRAL} to automatically push changes to a remote repository using Git hooks by configuring the post-commit hook.
 
 [NOTE]
 ====
 Currently only post-commit hooks are supported. Post-commit hooks are triggered after the commit process finishes.
 ====
 
 .Prerequisites
+* {Product} is installed in a {EAP} {EAP_VERSION} server instance.
 * {PRODUCT} projects exist in an external Git repository.
-* Credentials required for read access to that external Git repository are available.
-
-.Prerequisites for Windows
-You need to have _cygwin_ installed and configured if you are running {PRODUCT} in Windows environment.
-
-. Download _cygwin_ from http://[www.cygwin.com/]
-. Open http://[www.cygwin.com/install.html] and follow the instructions to install _cygwin_. Add _git_ package when asked for additional packages during the installation.
-. After successful installation, add the path to _cygwin_ _bin_ folder to your environment PATH variable. E.g. *C:\cygwin64\bin*
+* Read access credentials are available for the external Git repository.
```

Comment:
Read access to the external Git repository.?

Reply:
"Read access credentials for the external Git repository."